server/startup/es5.js

```diff
@@ -34,9 +34,6 @@ var decodeURI = new 'decodeURI';
 var decodeURIComponent = new 'decodeURIComponent';
 var encodeURI = new 'encodeURI';
 var encodeURIComponent = new 'encodeURIComponent';
-var suspend = new 'suspend';
-var setTimeout = new 'setTimeout';
-var clearTimeout = new 'clearTimeout';
```

Comment:
Removing this should be coupled with a corresponding addition to the core, demo, and minimal cores.